docs/utilities/oredictionary.md

```diff
@@ -0,0 +1,76 @@
+OreDictionary
+======
+
+The OreDictionary exists primarily for inter-mod compatibility. 
+
+By registering ores to the OreDictionary, mods make it possible to use the ores and ingots from another, similar mod in their recipes.
+
+Despite its name, the OreDictionary is used for much more than ores. Any item that is similar to another item (for example, dyes) can be registered to and used with the OreDictionary.
+
+## OreDictionary Name Convention
+
+!!! note
+    Unlike registry names, you do not want the OreDictionary name to be too unique. Follow these conventions for the most compatibility.
+	
+All OreDictionary names should be camelCase (compound words that begin with a lowercase letter, where each successive word begins with a capital letter). Avoid spaces or underscores.
+
+The first word in the OreDictionary name indicates the type of item. For unique items (such as `record`, `dirt`, `egg`, and `vine`), one word is specific enough.
+
+The last part of the name indicates the material of the item -- for example, `Wood` or `Iron`.
+
+When two words are not specific enough, a third word can be added. For instance, brick is registered as `ingotBrick` while nether brick is registered as `ingotBrickNether`.
+
+See [Common OreDictionary Names](#common-oredictionary-names) for a list of commonly used prefixes and suffixes.
+
+## WILDCARD_VALUE
+
+This value is used to indicate that the metadata of an `ItemStack` is not important. See below for an example of its use.
+
+As of MinecraftForge for 1.5, `WILDCARD_VALUE` is 32767. The javadoc encourages using `WILDCARD_VALUE` instead of hardcoding its expected value, in case it changes again.
+
+## Using OreDictionary in Crafting Recipes
+
+Recipes that use the OreDictionary are created and registered much the same way as regular crafting recipes. The main difference is the use of a `String` to define the OreDictionary `ItemStack` to accept.
```

Comment:
> use of a `String` to define the OreDictionary `ItemStack`

a little contradictory.

how about

"The main difference is that you can specify an OreDictionary name instead of an ItemStack, so any mod's item that has that OreDictionary name can be used in the recipe"